features/0037-present-proof/README.md

```diff
@@ -165,8 +166,41 @@ The optional `value`, when present, holds the value of the attribute to reveal i
 An attribute specification must specify a `value`, a `cred_def_id`, or both: 
 
 * if `value` is present and `cred_def_id` is absent, the preview proposes a self-attested attribute;
-* if `value` and `cred_def_id` are both present, the preview proposes verifiable claim to reveal in the presentation;
-* if `value` is absent and `cred_def_id` is present, the preview proposes verifiable claim not to reveal in the presentation.
+* if `value` and `cred_def_id` are both present, the preview proposes a verifiable claim to reveal in the presentation;
+* if `value` is absent and `cred_def_id` is present, the preview proposes a verifiable claim not to reveal in the presentation.
+
+##### Referent
+
+The optional `referent` can be useful in specifying multiple-credential presentations. Its value indicates which credential 
+will supply the attribute in the presentation. Sharing a `referent` value between multiple attribute specifications indicates that the holder's same credential supplies the attribute.
+
+Any attribute specification using a `referent` must also have a `cred_def_id`; any attribute specifications sharing a common `referent` value must all have the same `cred_def_id` value.
```

Comment:
Why? Wouldn't the more logical requirement be that there must be at least one cred_def_id value across all items with a common referent--and that all of them that do appear must be consistent? In other words, if I used the same referent 6 times, but only mentioned the cred_def_id once, wouldn't that be good enough?

What happens when this isn't an Indy-style cred, so it lacks a cred def? Couldn't other cred types use a credential id (not cred *def* id) as the thing they stitch together?

Reply:
Note that the holder already has credentials from which he creates the proposal, so the holder necessarily knows the cred def id. Elsewhere in the RFC, recall that skipping the cred def id marks the attribute as self-attested. So the other 5 attributes would count as self-attested here. Self-attested attributes don't come from a credential, unless I've completely spaced on something fundamental.

If it wasn't clear to you, it won't be clear to anyone else: I will amend the wording to be explicit here, and submit via this PR, Monday.

Reply:
This explanation makes perfect sense. Thank you. I agree that a clarifying sentence or two would be helpful.

Reply:
Updated accordingly.